src/main/java/com/powsybl/openloadflow/ac/AcLoadFlowContext.java

```diff
@@ -33,6 +34,14 @@ public AcLoadFlowContext(LfNetwork network, AcLoadFlowParameters parameters) {
         super(network, parameters);
     }
 
+    @Override
+    public JacobianMatrix<AcVariableType, AcEquationType> getJacobianMatrix() {
```

Comment:
Why do you need the two overrides?

Reply:
When load model are simulated, Jacobian matrix depends on target p & q of loads, so if a load target is updated we need to invalidate the matrix.